### PR TITLE
Odin landing page

### PR DESCRIPTION
**Pull request by Gbemisola19**
## Built with:
* ### Html elements,
  - `div`, `unordered list(ul)`, `button`
* ### CSS,
  - `display`, `width`, `height` `margin`, `padding`, `font-size`
  - `border-radius`, `color`, `background-color`, `font-style`
* ### Linters
  - [Prettier](https://prettier.io/)
  - [stylelint](https://stylelint.io/)
* ### Acknowledgement
   - [project link](https://www.theodinproject.com/lessons/foundations-landing-page)

### CODE REVIEW

**Review by OMOTAYO05 — COMMENTED** [review]
deployment checked, html,css and readme checked

File: README.md [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+- Html elements,
+  - `div`, `unordered list(ul)`, `button`
+- CSS,
+  - `display`, `width`, `height` `margin`, `padding`, `font-size`
+  - `border-radius`, `color`, `background-color`, `font-style`
+- Linters
+  - [Prettier](https://prettier.io/)
+  - [stylelint](https://stylelint.io/)
 
 ## Live Demo
```
Comment (teckcipher) [inline README.md:21]: Before merging, let's add the correct lint for the live demo (production).

File: README.md
Comment (teckcipher) [inline README.md:1]: Let's update `## 👤 Author` and `## Acknowledgments` and other sections before merging.

**Review by teckcipher — APPROVED** [review]
Kindly make the suggested changes before merging.

Add Odin assignment link to your PR description and readme (acknowledgement section) to your README.

**Pull request merged**